### PR TITLE
fixes #55

### LINKED ISSUE #55: All python notes fail on `create_version()`
There is currently no logic to handle python notes in `create_version()`

This causes all python notes builds to fail.

### PR DESCRIPTION
**Pull request by jwalton3141**
Addresses issue #55 

In a new PR to satiate @csgillespie

### CODE REVIEW

File: R/get_logo_path.R [hunk 1]
```diff
@@ -93,7 +93,10 @@ create_version = function() {
     pkg_name = get_r_pkg_name()
     pkg_ver = packageVersion(pkg_name)
   } else {
-    pkg_ver = ""
+    pkg_name = get_python_pkg_name()
```
Comment (csgillespie) [inline R/get_logo_path.R:96]: Technically, there may be more than one package, so `get_python_pkg_name()[1]`

**Review by csgillespie — CHANGES_REQUESTED** [review]

File: R/get_logo_path.R (hunk 1 above)
Comment (jwalton3141) [reply R/get_logo_path.R:96]: Sure. Should the same change be made to `get_r_pkg_name()` -> `get_r_pkg_name()[1]` as the same is also true for `r`?

**Comment by csgillespie** [comment]
Yep good spot

**Comment by jwalton3141** [comment]
_re._ `pip` vs `pip3` as mentioned in the original PR.

Leaving as `pip` is fine. As the checks are performed in `make final`, the virtual environment is activated, which we explicitly specify as python3. As such, `pip == pip3`.

**Review by csgillespie — APPROVED** [review]

**Pull request merged**